src/util/change-types.ts

```diff
@@ -37,6 +38,18 @@ type StationsWithoutNameOffsetAttributes =
     | LondonTubeBasicStationAttributes
     | ShanghaiSuburbanRailwayStationAttributes
     | OsakaMetroStationAttributes;
+const stationTypesWithTransfer = [
```

Comment:
Generally we check the key on the fly to determine whether it supports transfer or color.

This should be a convention that these two keys are reserved.

Listing as const variable will cause additional maintenance and should be avoid by default.